### ISSUE TITLE
Unable to access 'mysql' database in myduckserver

### ISSUE THREAD

**Issue by earayu** [apecloud/myduckserver#239]
## Description
When trying to access the 'mysql' database in myduckserver, the system shows error "database not found: mysql" despite the database being listed in `show databases` output.

## Reproduce steps
1. Start myduckserver container:
```bash
docker run -p 13306:3306 -p 15432:5432 apecloud/myduckserver:latest
```

2. Connect to server and verify databases:
```sql
mysql -h127.0.0.1 -P13306 -uroot

mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| information_schema |
| main               |
| mysql              |
| performance_schema |
| public             |
+--------------------+
5 rows in set (0.01 sec)
```

3. Try to access mysql database:
```sql
mysql> use mysql;
ERROR 1105 (HY000): unknown error: database not found: mysql
```

4. Direct connection attempt also fails:
```sql
mysql -h127.0.0.1 -P13306 -uroot -Dmysql
ERROR 1105 (HY000): unknown error: database not found: mysql
```

**Comment by TianyuZhang1214** [comment]
@earayu 
Thank you for your feedback! We'll make sure to notify you as soon as the issue is resolved.

**Comment by TianyuZhang1214** [comment]
Hi @earayu ,

We've resolved the issue in #242. Please give it another try and let us know if you encounter any further problems. Thank you!
